File: Backend/WebAPI/Requests/SessionRequest.cs
using System;
using System.Collections.Generic;

namespace WebAPI.Requests
{
    public class SessionRequest
    {
        public int FilmId { get; set; }
        public int HallId { get; set; }
        public DateTime StartDateTime { get; set; }
        public IReadOnlyCollection<SeatTypePriceRequest> SeatTypePrices { get; set; } = null!;
    }
}

File: Frontend/src/app/Requests/SessionRequest.ts
import { SeatTypePriceRequest } from "./SeatTypePriceRequest";

export class SessionRequest {
    filmId : number;
    hallId : number;
    startDateTime : Date;
    seatTypePrices : SeatTypePriceRequest[];
    
    constructor(
        filmId : number,
        hallId : number,
        startDateTime : Date,
        seatTypePrices : SeatTypePriceRequest[]
    ) {
        this.filmId = filmId;
        this.hallId = hallId;
        this.startDateTime = startDateTime;
        this.seatTypePrices = seatTypePrices;
    }
}